src/modules/esl-utils/dom/test/rtl.test.ts

```diff
@@ -22,6 +22,20 @@ describe('RTLUtils', () => {
       gcsSpy.mockImplementationOnce(() => ({direction: 'rtl'} as CSSStyleDeclaration));
       expect(RTLUtils.isRtl(el)).toBe(true);
     });
+
+    test('normalizeScrollLeft', () => {
```

Comment:
The outside effect `RTLUtils.scrollType` of `normalizeScrollLeft` unit is not covered.